beacon-chain/rpc/eth/beacon/api_structs.go

```diff
@@ -0,0 +1,57 @@
+package beacon
+
+type Phase0ProduceBlockV3Response struct {
+	Version                 string       `json:"version" validate:"required"`
```

Comment:
As stated in another place, I would not add `validate` to responses nor would I validate them. A validation error will return `400 Bad Request`, but it's not the user's request that was incorrect.